### PR TITLE
Add crate display to summary and improve hover text

### PR DESCRIPTION
**Pull request by amandabacon**

### CODE REVIEW

**Comment by tlatorre-uchicago** [comment]
@amandabacon is this ready to go or are you still working on it?

**Comment by amandabacon** [comment]
@tlatorre-uchicago I am working on the HV Status display that Erica pointed out. However, if you want to run this version, be my guest. I do not know when I will finish the HV Status code.

File: minard/templates/state.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% macro collapsible_header(title,name) %}
-<div class="row">
+<div class="row col-md-12">
```
Comment (tlatorre-uchicago) [inline minard/templates/state.html]: You shouldn't mix the row and column classes here.

Comment (amandabacon) [inline minard/templates/state.html]: Good eye! I will fix it.

**Comment by tlatorre-uchicago** [comment]
merged.

**Pull request merged**